docs/source/configuration.rst

```diff
@@ -230,13 +234,36 @@ projects. In particular it provides mock objects for:
   returns nothing.
 
 .. note::
-
     If there are other builtin macros which would make your life easier,
     consider submitting the idea (or even better a pull request) on `github`_.
 
 .. _`dbt`: https://www.getdbt.com/
 .. _`github`: https://www.github.com/sqlfluff/sqlfluff
 
+Dbt Project Configuration
+-------------------------
+
+dbt is not the default templater for *sqlfluff* (it is Jinja). In order
+to get started using *sqlfluff* with a dbt project you will need the following
+configuration:
+
```

Comment:
Can we add a brief line or paragraph here on the performance implications and how to choose the right templater? My assumption here is that the choice between the `jinja` and `dbt` templaters is that there's a tradeoff between macro and function support and performance and simplicity. I'd love to help people make a more informed choice about which templater to use and why.